tests/comparisons/Model/testBakeEnum.php

```diff
@@ -0,0 +1,18 @@
+<?php
+declare(strict_types=1);
+
+namespace Bake\Test\App\Model\Enum;
+
+/**
+ * FooBar Enum
+ */
+enum FooBar: int
```

Comment:
`string` return type might be better here. Personally I prefer my raw db values to be readable too. We could default to`string ` and have an option to specify the return type of required.

Reply:
:+1: to defaulting to string enums. They are far simpler to operate over strings. You can always optimize the storage of stable enums by using concrete enum columns as well.

Reply:
Can we add a flag --backed ?
I prefer them for various reasons

Reply:
I think an option is needed here. `--backing string` makes the most sense

Reply:
I'd image people will be so confused and mix up `bake` and `backing`

Reply:
--backed-enum ?

Reply:
> I'd image people will be so confused and mix up bake and backing

You need to be really bad a English to mixed up those.

> --backed-enum

The "enum" sounds redundant

How about `--backing-type`?